docs/reference/indices/list_query.rst

```diff
@@ -1,7 +1,7 @@
 Querying a List Index
 =====================
 
-.. automodule:: gpt_index.indices.query.list
+.. automodule:: gpt_index.indices.list
```

Comment:
so this would show both the index and query class? 

(it's actually fine, but there will be duplicates with reference/indices/list.rst)